### DISCUSSION TITLE
When/where was the world's largest recorded earthquake?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#727] [Galaxy Trivia] [answered]
**Bonus:** What did it measure on the Richter scale?

**Comment by SANKARAMDAS** [comment]
Valdivia, Chile
On May 22, 1960 a great Mw 9.5(Richter scale) earthquake, the largest earthquake ever instrumentally recorded, that struck near Valdivia, Chile, occurred off the coast of southern Chile. This earthquake generated a tsunami that was destructive not only along the coast of Chile, but also across the Pacific in Hawaii, Japan, and the Philippines.

**Comment by peterbaccaro** [comment, marked as answer]
The largest recorded earthquake in the world occurred on **May 22, 1960**, off the coast of southern Chile. Known as the "Valdivia Earthquake" and the "Great Chilean Earthquake" it had a magnitude of **9.5 on the moment magnitude scale**, making it the most powerful earthquake ever recorded.  Moment magnitude estimates are about the same as Richter magnitudes for small to large earthquakes. But only the moment magnitude scale is capable of measuring M8 (read "magnitude 8") and greater events accurately. Magnitudes are based on a logarithmic scale (base 10).

The earthquake struck at 7:11 pm local time and lasted for approximately 10 minutes. **The epicenter of the earthquake was located in the Pacific Ocean, about 100 miles (160 km) off the coast of Chile**. The quake triggered a series of devastating tsunamis that caused widespread damage and loss of life along the Chilean coast and in other Pacific countries, including Hawaii, Japan, and the Philippines.
The earthquake was named after the nearby city of Valdivia, which was one of the hardest hit areas. 

The Great Chilean Earthquake was caused by the subduction of the Nazca plate beneath the South American plate, along the Peru-Chile Trench. The seismic activity caused by this subduction zone is responsible for the formation of the Andes Mountains and the high level of seismic activity in the region.

Since the Great Chilean Earthquake, several other earthquakes of magnitude 9.0 or higher have occurred around the world, including the 2004 Indian Ocean earthquake and tsunami and the 2011 Tohoku earthquake and tsunami in Japan. However, the Great Chilean Earthquake remains the most powerful earthquake ever recorded in modern history.
